### ISSUE TITLE
.rb file permissions in packaged rubygem

### ISSUE THREAD

**Issue by pbevin-tc** [kigster/attr_memoized#1]
I use `attr_memoized` in my production build, but the `.rb` files are unreadable in the 0.3.0 rubygem because they get installed (by `root` in my case) with mode 0640.

```
-rw-r----- 1 root root 8612 Sep  3 20:21 lib/attr_memoized.rb
-rw-r----- 1 root root   75 Sep  3 20:21 lib/attr_memoized/version.rb
```

My app runs as `nobody`, and fails on `require "attr_memoized"` with:

```
/app/lib/init.rb:3:in `require': cannot load such file -- attr_memoized (LoadError)
```

**Comment by kigster** [comment]
Thanks for letting me know, I'll fix it in 0.3.1

**Comment by kigster** [comment]
Please update to `0.3.1`. Thanks!
